src/rpc/misc.cpp

```diff
@@ -142,6 +143,93 @@ static UniValue createmultisig(const JSONRPCRequest& request)
     return result;
 }
 
+UniValue deriveaddresses(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.empty() || request.params.size() > 3)
+        throw std::runtime_error(
+            RPCHelpMan{"deriveaddresses", {
+                "\nDerives one or more addresses corresponding to an output descriptor.\n"
+                "Examples of output descriptors are:\n"
+                "    pkh(<pubkey>)                        P2PKH outputs for the given pubkey\n"
+                "    wpkh(<pubkey>)                       Native segwit P2PKH outputs for the given pubkey\n"
+                "    sh(multi(<n>,<pubkey>,<pubkey>,...)) P2SH-multisig outputs for the given threshold and pubkeys\n"
+                "    raw(<hex script>)                    Outputs whose scriptPubKey equals the specified hex scripts\n"
+                "\nIn the above, <pubkey> either refers to a fixed public key in hexadecimal notation, or to an xpub/xprv optionally followed by one\n"
+                "or more path elements separated by \"/\", where \"h\" represents a hardened child key.\n"
+                "For more information on output descriptors, see the documentation in the doc/descriptors.md file.\n"
+            }, {
+                {"descriptor", RPCArg::Type::STR, /* opt */ false, /* default_val */ "", "The descriptor."},
+                {"begin", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "", "If a ranged descriptor is used, this specifies the beginning of the range to import."},
+                {"end", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "", "If a ranged descriptor is used, this specifies the end of the range to import."}
+            }}.ToString() +
```

Comment:
Compile failure can be fixed with (large diff due to whitespace fixes)

```diff
diff --git a/src/rpc/misc.cpp b/src/rpc/misc.cpp
index 12d165fff7..f2adec797e 100644
--- a/src/rpc/misc.cpp
+++ b/src/rpc/misc.cpp
@@ -145,30 +145,27 @@ static UniValue createmultisig(const JSONRPCRequest& request)
 
 UniValue deriveaddresses(const JSONRPCRequest& request)
 {
-    if (request.fHelp || request.params.empty() || request.params.size() > 3)
+    if (request.fHelp || request.params.empty() || request.params.size() > 3) {
         throw std::runtime_error(
-            RPCHelpMan{"deriveaddresses", {
-                "\nDerives one or more addresses corresponding to an output descriptor.\n"
-                "Examples of output descriptors are:\n"
-                "    pkh(<pubkey>)                        P2PKH outputs for the given pubkey\n"
-                "    wpkh(<pubkey>)                       Native segwit P2PKH outputs for the given pubkey\n"
-                "    sh(multi(<n>,<pubkey>,<pubkey>,...)) P2SH-multisig outputs for the given threshold and pubkeys\n"
-                "    raw(<hex script>)                    Outputs whose scriptPubKey equals the specified hex scripts\n"
-                "\nIn the above, <pubkey> either refers to a fixed public key in hexadecimal notation, or to an xpub/xprv optionally followed by one\n"
-                "or more path elements separated by \"/\", where \"h\" represents a hardened child key.\n"
-                "For more information on output descriptors, see the documentation in the doc/descriptors.md file.\n"
-            }, {
-                {"descriptor", RPCArg::Type::STR, /* opt */ false, /* default_val */ "", "The descriptor."},
-                {"begin", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "", "If a ranged descriptor is used, this specifies the beginning of the range to import."},
-                {"end", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "", "If a ranged descriptor is used, this specifies the end of the range to import."}
-            }}.ToString() +
-            "\nResult:\n"
-            "[ address ] (array) the derived addresses\n"
-            "\nExamples:\n"
-            "\nFirst three native segwit receive addresses\n"
-            + HelpExampleCli("deriveaddresses", "\"wpkh([d34db33f/84h/0h/0h]xpub6DJ2dNUysrn5Vt36jH2KLBT2i1auw1tTSSomg8PhqNiUtx8QX2SvC9nrHu81fT41fvDUnhMjEzQgXnQjKEu3oaqMSzhSrHMxyyoEAmUHQbY/0/*)\" 0 2") +
-            ""
-        );
+            RPCHelpMan{"deriveaddresses",
+                {"\nDerives one or more addresses corresponding to an output descriptor.\n"
+                 "Examples of output descriptors are:\n"
+                 "    pkh(<pubkey>)                        P2PKH outputs for the given pubkey\n"
+                 "    wpkh(<pubkey>)                       Native segwit P2PKH outputs for the given pubkey\n"
+                 "    sh(multi(<n>,<pubkey>,<pubkey>,...)) P2SH-multisig outputs for the given threshold and pubkeys\n"
+                 "    raw(<hex script>)                    Outputs whose scriptPubKey equals the specified hex scripts\n"
+                 "\nIn the above, <pubkey> either refers to a fixed public key in hexadecimal notation, or to an xpub/xprv optionally followed by one\n"
+                 "or more path elements separated by \"/\", where \"h\" represents a hardened child key.\n"
+                 "For more information on output descriptors, see the documentation in the doc/descriptors.md file.\n"},
+                {{"descriptor", RPCArg::Type::STR, /* opt */ false, /* default_val */ "", "The descriptor."},
+                    {"begin", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "", "If a ranged descriptor is used, this specifies the beginning of the range to import."},
+                    {"end", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "", "If a ranged descriptor is used, this specifies the end of the range to import."}},
+                RPCResult{
+                    "[ address ] (array) the derived addresses\n"},
+                RPCExamples{
+                    "\nFirst three native segwit receive addresses\n" + HelpExampleCli("deriveaddresses", "\"wpkh([d34db33f/84h/0h/0h]xpub6DJ2dNUysrn5Vt36jH2KLBT2i1auw1tTSSomg8PhqNiUtx8QX2SvC9nrHu81fT41fvDUnhMjEzQgXnQjKEu3oaqMSzhSrHMxyyoEAmUHQbY/0/*)\" 0 2")}}
+                .ToString());
+    }
 
     RPCTypeCheck(request.params, {UniValue::VSTR, UniValue::VNUM, UniValue::VNUM});
     const std::string desc_str = request.params[0].get_str();

Reply:
Thanks, done.